content/en/blog/_posts/2023-05-04-statefulset-autodelete.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: blog
+title: 'Kubernetes 1.27: StatefulSet PVC Auto-Deletion (beta)'
+date: 04-05-2023
```

Comment:
Should this be 05-04-2023 (May 04)?

Reply:
switched to year-first date so there's no ambiguity